projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "event_queue.h"
+#include "status.h"
+
+typedef enum { LIGHTS_BOARD_FRONT, LIGHTS_BOARD_REAR } BoardType;
+
+StatusCode lights_gpio_init(BoardType boardtype);
+StatusCode lights_gpio_set(Event e);
+StatusCode get_board_type(BoardType* type);
```

Comment:
Pointers should be right aligned. (I'm planning to fix our `.clang-format` file to better enforce this with one of my open PRs).